org.lflang/src/org/lflang/generator/ReactionInstanceGraph.java

```diff
@@ -27,16 +27,20 @@ STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
 package org.lflang.generator;
 
 import java.util.ArrayList;
+import java.util.HashSet;
 import java.util.LinkedHashSet;
 import java.util.List;
 import java.util.Set;
+import java.util.Stack;
 import java.util.stream.Collectors;
 
 import org.lflang.generator.ReactionInstance.Runtime;
 import org.lflang.generator.c.CUtil;
 import org.lflang.graph.PrecedenceGraph;
 import org.lflang.lf.Variable;
 
+import kotlin.Pair;
```

Comment:
Do we really need a dependence on kotlin here? This precludes compiling in Eclipse, and this extra dependence looks out of proportion with needing a pair.